packages/ui/src/elements/ListControls/index.tsx

```diff
@@ -193,6 +197,25 @@ export const ListControls: React.FC<ListControlsProps> = (props) => {
                 {t('general:sort')}
               </Pill>
             )}
+            {listControlsMenu != null && listControlsMenu !== false && (
```

Comment:
We can simplify the `Array.isArray(listControlsMenu)`, ignore if you get type errors.

Reply:
Updated